<documentation title="Strict Types Declaration">
    <standard>
        <![CDATA[
    When declaring strict types you should not put whitespace around the opening bracket or before the closing bracket.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: No spaces around brackets.">
            <![CDATA[
declare(strict_types=1);
]]>
        </code>
        <code title="Invalid: Spaces around brackets.">
            <![CDATA[
declare( strict_types=1 );
]]>
        </code>
    </code_comparison>
</documentation>
